openedx/core/djangolib/model_mixins.py

```diff
@@ -12,3 +12,25 @@ def __init__(self, *args, **kwargs):  # pylint: disable=unused-argument
         Override to kill usage of this model.
         """
         raise TypeError("This model has been deprecated and should not be used.")
+
+
+class DeletableByUserValue(object):
+    """
+    This mixin allows inheriting models to delete instances of the model
+    associated with some user.
```

Comment:
#NIT not `some user`, should be `the specified user`

Reply:
I think these are synonymous.